src/cli/cli.go

```diff
@@ -61,13 +61,16 @@ func preRun(cc *cobra.Command, args []string) error {
 		print.Info(ctx, "Logging to file: "+logger.LogFile)
 	}
 
-	avoidTheseCommands = []string{
-		"help for corso",
```

Comment:
How about just replacing this to `Help about any command`.

Reply:
Done